docs/examples/nep-5.md

```diff
@@ -0,0 +1,473 @@
+---
+id: nep5
+title: NEP-5
+---
+
+> You can refer to **Precondition** part to check the initiation of `apiProvider`, `myWallet` and `myAccount`
+
+## Get Balance
+
+### With RPC Query
+
+```javascript
+const { default: Neon, nep5, rpc } = require("@cityofzion/neon-js");
+
+const generator = nep5.abi.balanceOf(
+  "80de34fbe3e6488ce316b722c5455387b001df31",
+  myAccount.address
+);
+const builder = generator();
+const script = builder.str;
+ 
+// local invocation
+rpc.Query.invokeScript(script)
+ .execute("http://rpc.url:portNum")
+ .then(res => {
+    console.log(res);
+ })
+ .catch(config => {
+    console.log(config);
+ });
+```
+
+
+
+## Send NEP-5
+
+### With Neon API (High Level)
+
+```javascript
+const { default: Neon, api, sc } = require("@cityofzion/neon-js");
+
+// we must change data type of contract parameters
+const param_sending_address = sc.ContractParam.byteArray(
+  myAccount.address,
+  "address"
+);
+const param_receiving_address = sc.ContractParam.byteArray(
+  "AN6nd3B7iQxKK23DWAFSzgykbyTjMdieXD",
+  "address"
+);
+// we need to mul 1e8 because the format is Fixed8
+const param_amount = Neon.create.contractParam("Integer", 100 * 1e8);
```

Comment:
This is dangerous if the nep5 token is not using 8 decimals places. Why are we not using the nep5 abi function for transfer?

Reply:
We may need to add nep5 abi function as high level method. @WafeiBaBa

Reply:
Now we use nep5 abi sc builder to generate script in high level method, and build param step by step in low level and raw level methods